tests/integration/enc/models/test_alexnet.py

```diff
@@ -8,12 +8,10 @@
 
 
 class TestAlexNetMultiLayerEncoder:
-    def test_smoke(self, subtests):
+    def test_smoke(self):
         multi_layer_encoder = enc.alexnet_multi_layer_encoder(pretrained=False)
         assert isinstance(multi_layer_encoder, enc.alexnet.AlexNetMultiLayerEncoder)
-
-        with subtests.test("repr"):
-            assert isinstance(multi_layer_encoder, enc.alexnet.AlexNetMultiLayerEncoder)
+        assert isinstance(multi_layer_encoder, enc.alexnet.AlexNetMultiLayerEncoder)
```

Comment:
It would be better to split this into two tests, e.g. `test_main_smoke` and `test_repr_smoke`

Reply:
On that note: the second line is completely redundant :facepalm: It should be something like

```python
assert isinstance(repr(multi_layer_encoder), str)
```

to check if the `repr` can be constructed and is a string.